credits.txt

```diff
@@ -198,6 +198,7 @@ contributed to Endless Sky:
   jozef-mitro
   Jugosloven1612
   Karirawri
+  kestrel1110 (Bereskatuket)
```

Comment:
All the names under this part of the credits are GitHub `@`s. It seems like it'd be a bit cluttered if we started allowing people to put multiple names, especially because it could easily lead to the credits overflowing the UI.

Reply:
Yeah I think I'll do that, it's just that I've always gone by Bereskatuket at my main name so it seemed appropriate.
But I do recognize that there should be more organization, so I'll do this.